modules/optimizing-mtu-networking.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-networking.adoc
+
+[id="optimizing-mtu_{context}"]
+= Optimizing the MTU for your network
+
+There are two important maximum transmission units (MTUs): the network interface card (NIC) MTU and the SDN overlay's MTU.
+
+The NIC MTU must be less than or equal to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
+
+The SDN overlay's MTU must be less than the NIC MTU by 50 bytes at a minimum. This accounts for the SDN overlay header. So, on a normal ethernet network, set this to `1450`. On a jumbo frame ethernet network, set this to `8950`.
+
+[NOTE]
+====
+This 50 byte overlay header is relevant to the OpenShift SDN. Other SDN solutions might require the value to be more or less.
+====
+
+.Procedure
```

Comment:
So I'd keep the MTU discussion possibly, but omit the procedure and note that this must happen during cluster installation only.

And the MTU size varies between OpenShift SDN and OVN-Kubernetes; We now offer both as of 4.6 GA.